backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -35,4 +44,33 @@ class CardMutationService {
         }
         return true
     }
+
+    @GraphQLDescription("Activate a digital EAK")
+    fun activateCard(
+        project: String,
+        card: CardGenerationModel,
+        overwrite: Boolean
+    ): CardActivationResultModel {
+        if (card.codeType == CodeType.static) {
+            return CardActivationResultModel(ActivationState.failed)
+        }
+
+        val cardHash = Base64.getDecoder().decode(card.cardInfoHashBase64)
+        val activationSecret = Base64.getDecoder().decode(card.activationSecretBase64)
+        val activatedCard = transaction { CardRepository.findByHashModelAndActivationSecret(project, cardHash, activationSecret,) }
+
+        if (activatedCard == null || activatedCard.activationSecret == null) {
+            return CardActivationResultModel(ActivationState.failed)
+        }
+
+        if (!overwrite) {
```

Comment:
I think we're missing the check that the totp secret already exists:
```suggestion
        if (activatedCard.totpSecret != null && !overwrite) {
```